### PR TITLE
#999 responsive component display.

### PR DESCRIPTION
**Pull request by saadmhybris**
Closes sap/fundamental#999

Adds an option to display responsive component. any demo page can be displayed using this option. 

#### Test

* http://localhost:4000/layouts/responsive-test.html

#### Changelog

**New**

* responsive component display option

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit 08e18aa932df8f086d8df3980588e2bacf93c24b

https://deploy-preview-1030--fundamental.netlify.com

File: docs/_includes/demo-page-components/shell.html [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+<button class=" fd-button--shell sap-icon--bell" aria-label="Notifications">
+<span class="fd-counter fd-counter--notification" aria-label="Unread count">25</span></button>
+            </div>
+            <div class="fd-shellbar__action fd-shellbar__action--collapsible">
+
+<button class=" fd-button--shell sap-icon--pool" aria-label="Pool"></button>
+            </div>
+        <div class="fd-shellbar__action fd-shellbar__action--collapse">
+          <!--
```
Comment (ckoutsiaris) [inline docs/_includes/demo-page-components/shell.html]: Maybe this block can be removed?

File: docs/_includes/demo-page-components/shell.html [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+
+  <div class="fd-shellbar-collapse">
```
Comment (ckoutsiaris) [inline docs/_includes/demo-page-components/shell.html]: Maybe this block can be removed?

File: docs/_includes/demo-page-components/shell.html [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+      <li><a role="button" class="fd-menu__item">Application A</a>
+    </li>
+      <li><a role="button" class="fd-menu__item">Application B</a>
+    </li>
+      <li><a role="button" class="fd-menu__item">Application C</a>
+    </li>
+      <li><a role="button" class="fd-menu__item">Application D</a>
+    </li>
+    </ul>
+</nav>
```
Comment (ckoutsiaris) [inline docs/_includes/demo-page-components/shell.html]: Indentation is off?
I am using beautify plugin in vscode and it formats the html nicely with one shortcut. 
https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify

File: docs/_includes/demo-page-components/shell.html (hunk 2 above)
Comment (saadmhybris) [reply docs/_includes/demo-page-components/shell.html]: fixed

File: docs/_includes/demo-page-components/shell.html (hunk 3 above)
Comment (saadmhybris) [reply docs/_includes/demo-page-components/shell.html]: fixed

**Comment by saadmhybris** [comment]
@xak this is probably because the docs are not getting the latest styles compiled. 

pls try `bundle exec jekyll build` from the `docs` folder. 

the button renders fine on my end. 
screen shot 2018-11-28 at 9 55 30 am

**Comment by xak** [comment]
@saadmhybris This looks OK if you can fix the conflicts.

**Pull request merged**